test/e2e/overridepolicy_test.go

```diff
@@ -39,24 +39,29 @@ var _ = ginkgo.Describe("[OverridePolicy] apply overriders testing", func() {
 				Kind:       deployment.Kind,
 				Name:       deployment.Name,
 			},
-		}, policyv1alpha1.ClusterAffinity{
-			ClusterNames: framework.ClusterNames(),
-		}, policyv1alpha1.Overriders{
-			ImageOverrider: []policyv1alpha1.ImageOverrider{
-				{
-					Component: "Registry",
-					Operator:  "replace",
-					Value:     "fictional.registry.us",
-				},
-				{
-					Component: "Repository",
-					Operator:  "replace",
-					Value:     "busybox",
+		}, []policyv1alpha1.RuleWithCluster{
```

Comment:
This essentially removed the test coverage for the deprecated fields.
Even though they are deprecated but I still want the test to exist until we removed them from APIs.

Reply:
So I'll leave this alone for now and add the e2e case for OverrideRules separately later.